app/views/people/show.html.erb

```diff
@@ -26,14 +26,14 @@
             </figure>
           </div>
         <% end %>
-        <h1 <%= t_lang('document.contents') %>><%= t('document.contents') %></h1>
-        <nav role="navigation">
+        <nav role="navigation" aria-labelledby="document-contents-heading">
+          <h2 id="document-contents-heading" <%= t_lang('document.contents') %>><%= t('document.contents') %></h2>
           <ul>
             <li <%= t_lang('people.biography') %> >
               <%= link_to t('people.biography'), '#biography' %>
             </li>
             <% if @person.currently_in_a_role? %>
-              <li <%= t_lang('roles.heading', count: @person.current_role_appointments.count) %> >
```

Comment:
I think we might still need to pass `count: @person.current_role_appointments.count` as an option to `t_lang`?

Reply:
Ah, sorry! I've reverted that line now. Thanks for spotting that 👍